### PR TITLE
Create taylor-and-francis-council-of-science-editors-numeric.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/93725/citation-style-incorrect#latest

Sample paper: https://www.tandfonline.com/doi/pdf/10.1080/19490976.2021.1997294?needAccess=true

### CODE REVIEW

File: taylor-and-francis-council-of-science-editors-numeric.csl [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Taylor &amp; Francis - Council of Science Editors (numeric)</title>
+    <title-short>TF-CSE</title-short>
+    <id>http://www.zotero.org/styles/taylor-and-francis-council-of-science-editors-numeric</id>
+    <link href="http://www.zotero.org/styles/taylor-and-francis-council-of-science-editors-numeric" rel="self"/>
+    <link href="http://www.zotero.org/styles/taylor-and-francis-council-of-science-editors-numeric" rel="template"/>
```
Comment (adam3smith) [inline taylor-and-francis-council-of-science-editors-numeric.csl:8]: what's the actual template you used?

File: taylor-and-francis-council-of-science-editors-numeric.csl [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="by-cite">
```
Comment (adam3smith) [inline taylor-and-francis-council-of-science-editors-numeric.csl:140]: remove the author-date options

File: taylor-and-francis-council-of-science-editors-numeric.csl [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    <sort>
```
Comment (adam3smith) [inline taylor-and-francis-council-of-science-editors-numeric.csl:141]: this should sort by citation-number

File: taylor-and-francis-council-of-science-editors-numeric.csl [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+      <key variable="issued"/>
+      <key macro="author-short"/>
+    </sort>
+    <layout vertical-align="sup" delimiter=",">
+      <text variable="citation-number" vertical-align="baseline"/>
```
Comment (adam3smith) [inline taylor-and-francis-council-of-science-editors-numeric.csl:146]: I assume this shouldn't actually be baseline aligned

File: taylor-and-francis-council-of-science-editors-numeric.csl [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+    </layout>
+  </citation>
+  <bibliography et-al-min="12" et-al-use-first="10" second-field-align="flush">
+    <sort>
```
Comment (adam3smith) [inline taylor-and-francis-council-of-science-editors-numeric.csl:150]: remove sort section (so it sorts consecutively)

**Review by adam3smith — CHANGES_REQUESTED** [review]
Some clean-up from the author-date version left

File: taylor-and-francis-council-of-science-editors-numeric.csl [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text variable="title" form="short"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if match="any" variable="DOI">
+        <text variable="DOI" prefix="https://doi.org/"/>
```
Comment (adam3smith) [inline taylor-and-francis-council-of-science-editors-numeric.csl:51]: I'd stick with doi: I think, since that's what they want in the guide

**Comment by adam3smith** [comment]
Thanks! I think there is a dependent change (or addition?) that's good with this?

**Comment by POBrien333** [comment]
Yes, just created this: https://github.com/citation-style-language/journals/pull/124

**Pull request merged**